documentation/quarkus/quarkus-template.asciidoc

```diff
@@ -0,0 +1,122 @@
+:toc: macro
+toc::[]
+
+= Quarkus template
+
+https://code.quarkus.io/?g=org.devonfw&e=resteasy&e=resteasy-jackson&e=hibernate-validator&e=hibernate-orm&e=micrometer[Quarkus Code Generator] is providing a lot of alternatives on technologies and libraries to be integrated. Detailed guides to multiple topics can be found https://quarkus.io/guides/[here].
```

Comment:
@hohwille mentioned [here](https://github.com/devonfw/devon4j/pull/428#pullrequestreview-729712454l) we have some overlaps in our guides. The create a first devon4quarkus application guide also mentions the Quarkus Qode Generator and the Quarkus guides. Maybe one of us should reference the other guide.

Reply:
We decided to omit [theguide to create first devon4quarkus application](https://github.com/devonfw/devon4j/pull/428) and add some useful links directly here. @hohwille